<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>536</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_HoTen">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>88</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Họ và tên:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_CoQuan">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>DANH SÁCH SINH VIÊN</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_HoTen">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_NamSinh">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_NamSinh">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>88</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Năm sinh:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_DiaChi">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>88</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Địa chỉ:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_DiaChi">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_Luu">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>110</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Lưu</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_Exit">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>490</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>661</width>
     <height>192</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>200</number>
   </attribute>
   <column>
    <property name="text">
     <string>HoTen</string>
    </property>
    <property name="background">
     <color>
      <red>0</red>
      <green>0</green>
      <blue>0</blue>
     </color>
    </property>
   </column>
   <column>
    <property name="text">
     <string>NamSinh</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>DiaChi</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pButton_Xoa">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>170</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Xóa</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
